docs/resources/tutorials/core/qos/mutable/vulcanexus_change_mutable_qos/src/change_mutable_qos_pub1.cpp

```diff
@@ -0,0 +1,96 @@
+#include <memory>
+
+#include "rclcpp/rclcpp.hpp"
+#include "std_msgs/msg/string.hpp"
+
+#include "rmw_fastrtps_cpp/get_participant.hpp"
+#include "rmw_fastrtps_cpp/get_publisher.hpp"
+
+#include <fastdds/dds/publisher/qos/DataWriterQos.hpp>
+
+using namespace std::chrono_literals;
+
+class Node_ChangeMutableQoS_Pub1 : public rclcpp::Node
+{
+public:
+  Node_ChangeMutableQoS_Pub1()
+  : Node("node1_change_mutable_qos")
+  {
+    // Chatter publisher callback
+    auto publish =
+      [this]() -> void
+      {
+        msg_ = std::make_unique<std_msgs::msg::String>();
+        msg_->data = "Hello World: " + std::to_string(count_++);
+        RCLCPP_INFO(this->get_logger(), "PUB1 Publishing: '%s'", msg_->data.c_str());
+        pub_->publish(std::move(msg_));
+
+        eprosima::fastdds::dds::DataWriterQos dw_qos;
+        dw->get_qos(dw_qos);
+
+        eprosima::fastdds::dds::OwnershipStrengthQosPolicy dw_os_qos;
+        dw_os_qos = dw_qos.ownership_strength();
+        
+
+        RCLCPP_INFO(this->get_logger(), "Ownership strength: '%d'", dw_os_qos.value);
```

Comment:
This you could do in a single call. Maybe we can unify this and the previous log into a single one so the STDOUT output looks cleaner
```suggestion
        RCLCPP_INFO(this->get_logger(), "Ownership strength: '%d'", dw->get_qos().ownership_strength().value);
```

Reply:
Done